pkg/graph/schema.graphql

```diff
@@ -92,6 +92,24 @@ type CreateAccessibilityRequestPayload {
   userErrors: [UserError!]
 }
 
+enum AccessibilityRequestDeletionReason {
+  INCORRECT_APPLICATION_AND_LIFECYCLE_ID
+  NO_TESTING_NEEDED
+  OTHER
+
+  " For 508 requests that predate collecting this field "
+  UNKNOWN
```

Comment:
Do we gain value by using a different value for older records, or should we set all current records to `OTHER`?

Reply:
The users didn't choose other, so it might mean we are misrepresenting their situation. It's a product concern I think. @akostibas WDYT?

Reply:
For additional context, I _think_ we would be affecting impl data, but highly unlikely to affect prod data at this stage of the project.